src/codex_plus/llm_execution_middleware.py

```
@@ -363,61 +430,356 @@ async def process_request(self, request, path: str):
+                        async for chunk in response.aiter_bytes():
+                            chunk_count += 1
+                            logger.info(
+                                "📦 Received Cerebras chunk",
+                                extra={"chunk_index": chunk_count, "size": len(chunk) if chunk else 0},
+                            )
+                            if not chunk:
+                                continue
+
+                            # Log raw Cerebras SSE chunk
+                            try:
+                                log_dir = Path("/tmp/codex_plus/cerebras_responses")
```

Ruff (0.13.2)

Probable insecure usage of temporary file or directory: "/tmp/codex_plus/cerebras_responses"

(S108)